### PR TITLE
"Sub-10 millisecond latency on Java: Concurrent GC with Green Threads"

### PR DESCRIPTION
**Pull request by mtopolnik**

### CODE REVIEW

**Comment by hz-highfive** [comment]
The job `jet-oss-pr-builder-docs-no-build` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/251/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/251/jet-oss-pr-builder-docs-no-build-251.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
---------SUMMARY----------
--------------------------
There are unexpected files.
Unexpected file: hazelcast-jet-core/src/main/java/org
</details>
</pre>

File: hazelcast-jet-core/src/main/java/org [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/mtopol/dev/java/Benchmark/src/main/java/org
```
Comment (cangencer) [inline hazelcast-jet-core/src/main/java/org:1]: this seems some error?

**Comment by mtopolnik** [comment]
Yes, we'll add more advice to the Operations Guide.

**Comment by viliam-durina** [comment]
Maybe we should mention that not all processing is on green threads. On the other hand, most non-cooperative processors don't use high parallelism, like we don't have 96 kafka source processors normally.

**Pull request merged**